README.md

```diff
@@ -34,17 +37,25 @@ Loading this module through a script tag will make the ```IsIpfs``` obj availabl
 <script src="https://unpkg.com/is-ipfs/dist/index.js"></script>
 ```
 
-## Usage
+# Usage
 ```javascript
 const isIPFS = require('is-ipfs')
 
 isIPFS.multihash('QmYjtig7VJQ6XsnUjqqJvj7QaMcCAwtrgNdahSiFofrE7o') // true
 isIPFS.multihash('noop') // false
 
+isIPFS.multibase('bafybeie5gq4jxvzmsym6hjlwxej4rwdoxt7wadqvmmwbqi7r27fclha2va') // 'base32'
+isIPFS.multibase('zdj7WWeQ43G6JJvLWQWZpyHuAMq6uYWRjkBXFad11vE2LHhQ7') // 'base58btc'
+isIPFS.multibase('QmYjtig7VJQ6XsnUjqqJvj7QaMcCAwtrgNdahSiFofrE7o') // false (no multibase prefix in CIDv0)
+isIPFS.multibase('noop') // false
```

Comment:
This feels out of place to me. The rest of the API returns true/false. This is a simple call-through to `multibase.isEncoded`.

Reply:
Good point. Should it be changed to `true`/`false` or just removed? 
(I think I've added it only because `isIPFS.base32cid` introduced multibase as a dependency)

Reply:
Personally, I'd remove it. I considered that `isIPFS.base32cid` uses multibase as a dependency, and was thinking of suggesting to check like:

```js
typeof cid === 'string' && cid.slice(0, 4) === 'bafy' && isCID(cid)
```

...but js-cid depends on multibase so you're not saving any bundle bytes by depending on it and using it.